dashboard/app/controllers/api_controller.rb

```diff
@@ -206,7 +206,7 @@ def user_progress_for_stage
         script_level_id: script_level.try(:id),
         level_id: level.id,
         user_agent: request.user_agent,
```

Comment:
My inclination is that the non-UTF-8 characters are in the locale, not the user_agent. Should this not resolve the issue, I'll try the fix on user_agent. However, I'd prefer to not sanitize stuff needlessly.